projects/fsastorefront-e2e-cypress/cypress/helpers/changeRequest.ts

```diff
@@ -81,3 +83,84 @@ export function addTrailerLiability() {
       cy.get('.secondary-button').contains('Remove');
     });
 }
+
+export function startAddingDriverCheckout() {
+  cy.get('.fs-icon')
+    .should('have.length', 2)
+    //.eq(0)
+    .parentsUntil()
+    .contains(' Who or What Is Insured ')
+    .click({ force: true });
+  cy.get('.action-button').should('be.visible');
+  cy.get('.action-button').click();
+}
+
+export function checkAddDriverSteps() {
+  cy.get('h2').should('contain.text', ' Change Auto Insurance Policy ');
+  cy.get('.action-button').should('contain', 'Cancel');
+  cy.get('.progress-inner-wrapper')
+    .should('have.length', 2)
+    .within(() => {
+      cy.get('p.label').eq(0).contains('Driver Information');
+      cy.get('p.label').eq(1).contains('Change Preview');
+    });
+}
+
+export function checkDriverInformationStep() {
+  cy.get('cx-fs-cms-form-submit')
+    .should('be.visible')
+    .within(() => {
+      cy.get('h3').should('contain.text', 'Auto Information');
+    });
+  cy.get('[name="effectiveDate"]').should('have.value', todaysDateDriver);
+  cy.get('h4').should('contain.text', 'Additional Driver');
+}
+
+export function populateAdditionalDriverData() {
+  cy.get('cx-fs-cms-form-submit').within(() => {
+    cy.get('[name=firstName]').type('Sophie');
+    cy.get('[name="lastName"]').type('Digital');
+    cy.get('[name="dateOfBirth"]').type('1992-01-01');
+    cy.get('[name="driverGender"]').select('Not specified');
+    cy.get('[name="driverMaritalStatus"]').select('Single');
+    cy.get('[name="driverCategory"]').select('Occasional');
+    cy.get('[name="driverLicenceNumber"]').type('LN-88882');
+    cy.get('[name="driverLicenceDate"]').type('2019-02-02');
+  });
+}
+
+export function checkDrivers() {
+  cy.get('cx-fs-change-simulation').within(() => {
+    cy.get('.col-4').should('contain.text', 'John Moore');
+    cy.get('.col-4').should('contain.text', 'Phin Moore');
```

Comment:
Entire Moore family without the famous Donna :(

Reply:
nooo, Donna invited other family member to join FSA :)